### PR TITLE
Updates Adobe Experience Platform Assurance notes, references, and images in preparation for GA

### PR DESCRIPTION
**Pull request by caseymcj**
Updates Adobe Experience Platform Assurance notes, references, and images in preparation for GA.

In cases where the name "Adobe Experience Platform Assurance" is both contextually _and_ visually close to further references to it, the approved shortened name "Assurance" is substituted.

Uses of the name "Project Griffon" in historical contexts (i.e. release notes, etc) were not changed.

Additionally, the `aa-loop.gif` file was flattened / frozen / replaced to make it more consistent with other images within the documentation.

### CODE REVIEW

**Comment by caseymcj** [comment]
Question: do we want to add anything about quick-connect? Or should we leave the docs with only the QR code instructions?

File: beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-You may use the Validation Summary feature to quickly and easily manage JavaScript functions to validate events in a Project Griffon session. Each function receives the events in a Project Griffon session. You can write functions to validate your client configuration, event conditions, tests and use cases.
+You may use the Validation Summary feature to quickly and easily manage JavaScript functions to validate events in an Adobe Experience Platform Assurance session. Each function receives the events in an Assurance session. You can write functions to validate your client configuration, event conditions, tests and use cases.
 
 ## Getting Started
 
 To use this view, complete the following steps:
 
-1. [Setup Project Griffon](../set-up-project-griffon.md)
-2. [Create](./#creating-sessions) and [connect](./#connecting-to-a-session) to a Project Griffon session
+1. [Setup Adobe Experience Platform Assurance](../set-up-project-griffon.md)
```
Comment (calebk1) [inline beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md:11]: Same comment here - should be "Set up Adobe..."

**Review by calebk1 — COMMENTED** [review]
Looks good to me, but I just want to confirm one major things - with this change, is it moving out of beta? If so, we should remove beta mentions as well.

**Comment by caseymcj** [comment]
> Looks good to me, but I just want to confirm one major things - with this change, is it moving out of beta? If so, we should remove beta mentions as well.

Good question! We're not quite ready to remove the beta tag yet. That will be in another near-future PR.

File: beta/experience-platform-extension/experience-platform-debugging.md [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
 
-## Inspect Experience events with Project Griffon
+## Inspect Experience events with Adobe Experience Platform Assurance
 
-While the log messages can be quite helpful for an application developer, inspecting the events in the [Project Griffon UI](https://experience.adobe.com/griffon/) is even easier. In just a few steps, you can view the timestamp ordered list of all the events generated by the AEP Edge mobile extension, along with the confirmation that the request was successfully processed by the AEP Edge Network.
+While the log messages can be quite helpful for an application developer, inspecting the events in the [Assurance UI](https://experience.adobe.com/griffon/) is even easier. In just a few steps, you can view the timestamp ordered list of all the events generated by the AEP Edge mobile extension, along with the confirmation that the request was successfully processed by the AEP Edge Network.
```
[the remark this answers is not in the document]
Comment (caseymcj) [reply beta/experience-platform-extension/experience-platform-debugging.md:9]: The URLs are staying the same for now. I imagine that they'll change at some point though.

File: beta/experience-platform-extension/response-handling.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ For each Experience event that is sent through the AEP Edge extension, there can
 
 Each event handle that is returned by the Experience Edge Network is handled by the AEP Edge extension and it is dispatched as a new mobile SDK event.
 
-To see these events in a [Project Griffon session](https://experience.adobe.com/griffon%20), in **Search Events** type `Response event`.
+To see these events in an [Adobe Experience Platform Assurance session](https://experience.adobe.com/griffon%20), in **Search Events** type `Response event`.
```
[the remark this answers is not in the document]
Comment (caseymcj) [reply beta/experience-platform-extension/response-handling.md:9]: Yeah, that `%20` was there originally, so I didn't change it this time around (we're aiming to keep functionality untouched with this PR). That should be corrected when we update the URLs on a future PR.

The `%20` is the escape sequence for the whitespace character, so it's harmless to have at the end of a URL.

Good question though!

File: beta/project-griffon/README.md [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
 
-Project Griffon is a new, innovative beta product from [Adobe Experience Cloud](https://www.adobe.com/experience-cloud.html) to help you inspect, proof, simulate, and validate how you collect data or serve experiences in your mobile app.
+Adobe Experience Platform Assurance is a new, innovative beta product from [Adobe Experience Cloud](https://www.adobe.com/experience-cloud.html) to help you inspect, proof, simulate, and validate how you collect data or serve experiences in your mobile app.
 
-We're building Project Griffon to do the heavy lifting of getting your app implementation right, while you focus on creating engaging experiences.
+We're building Assurance to do the heavy lifting of getting your app implementation right, while you focus on creating engaging experiences.
 
 {% hint style="warning" %}
-Project Griffon is a beta product.
+Assurance is a beta product.
```
[the remark this answers is not in the document]
Comment (longAdobe) [reply beta/project-griffon/README.md:8]: Good point, does this PR represent the official "release" or will there be a separate PR to remove beta references?

File: beta/project-griffon/using-project-griffon/README.md [hunk 5]
```diff
@@ -16,7 +16,7 @@
 ![Creating a new session](../../../.gitbook/assets/assets_-m-julgvpg09f1jttuu__-m-kn_scoqogv3uegkzo_-m-koebjg71pqlppuurb_creat-new-session.gif)
 
 {% hint style="info" %}
-The Base URL is the root definition used to launch your app from a URL. A session URL is generated by which you may initiate the Project Griffon session. An example value might look like: `grifflab://default` In the **Base URL** field, type your app's base deep link definition.
+The Base URL is the root definition used to launch your app from a URL. A session URL is generated by which you may initiate the Assurance session. An example value might look like: `grifflab://default` In the **Base URL** field, type your app's base deep link definition.
```
[the remark this answers is not in the document]
Comment (longAdobe) [reply beta/project-griffon/using-project-griffon/README.md:19]: I recommend changing it. Probably best if we can represent a deep link url example without the project name, but if we can't come up with a good example, the project name should be fine.

**Review by praschetan — APPROVED** [review]

File: beta/project-griffon/using-project-griffon/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (caseymcj) [reply beta/project-griffon/using-project-griffon/README.md:19]: Here are a few ideas:
- `myapp://default`
- `assurance://default`
- `aeplab://default`

Any takers? Or any others?

File: beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md [hunk 6]
```diff
@@ -2,21 +2,21 @@
...
-You may use the Custom Validation feature to quickly and easily write a JavaScript function to validate events in a Project Griffon session.
+You may use the Custom Validation feature to quickly and easily write a JavaScript function to validate events in an Adobe Experience Platform Assurance session.
 
 ## Getting Started
 
 To use this view, complete the following steps:
 
-1. [Setup Project Griffon](../set-up-project-griffon.md)
-2. [Create](./#creating-sessions) and [connect](./#connecting-to-a-session) to a Project Griffon session
+1. [Setup Assurance](../set-up-project-griffon.md)
```
[the remark this answers is not in the document]
Comment (caseymcj) [reply beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md:11]: Good catch! I actually started to go down the path of correcting grammar as I went, but I soon realized (given the number of changes I was making) that grammar corrections should probably be a separate PR. I verified with Amber, and she confirmed that grammar is not my focus this time around. I actually saw a lot of places where we were using "setup" incorrectly (among other things). We'll probably want to address all of them together very soon.

Again, thanks for pointing this out. :)

File: beta/project-griffon/using-project-griffon/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (longAdobe) [reply beta/project-griffon/using-project-griffon/README.md:19]: my vote

myapp://default

File: SUMMARY.md [hunk 7]
```diff
@@ -94,7 +94,7 @@
   * [Analytics API reference](using-mobile-extensions/adobe-analytics/analytics-api-reference.md)
   * [Analytics event reference](using-mobile-extensions/adobe-analytics/analytics-event-reference.md)
   * [Products variable](using-mobile-extensions/adobe-analytics/product-variable.md)
-  * [Frequently asked questions](using-mobile-extensions/adobe-analytics/analytics-faq.md)
+    * [Frequently asked questions](using-mobile-extensions/adobe-analytics/analytics-faq.md)
```
[the remark this answers is not in the document]
Comment (caseymcj) [reply SUMMARY.md]: Thanks - no. I just removed it.

**Comment by caseymcj** [comment]
> Good point, does this PR represent the official "release" or will there be a separate PR to remove beta references?

I was presuming that we'd change this once Griffon officially moves to GA. But maybe these two events don't need to coincide...? If not, then I would be in favor of removing the beta notice too.

File: beta/project-griffon/using-project-griffon/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (caseymcj) [reply beta/project-griffon/using-project-griffon/README.md]: Thanks. I changed it to `myapp://default`.

File: beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md [hunk 8]
```diff
@@ -28,7 +28,7 @@ This feature allows you to write a function to validate events in your Project G
 | `uuid` | String | Universally unique identifier for the event. |
 | `timestamp` | Number | Timestamp from the device when the event was sent from the SDK. |
 | `eventNumber` | Number | Used to order when the event was sent. This key is useful when events have the same timestamp. |
-| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.griffon\). |
+| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.assurance\). |
```
Comment (ahmyr) [inline beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md:31]: For now, we aren't going to change our vendor name to com.adobe.assurance. So, let's leave this example as com.adobe.griffon for now so it matches what people will see.

File: beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md [hunk 9]
```diff
@@ -34,9 +34,9 @@ After you have tested your function, you can click **Upload** to save your valid
 | Key | Type | Description |
 | :--- | :--- | :--- |
 | `uuid` | String | Universally unique identifier for the event. |
-| `timestamp` | Number | Timestamp from the client when the event was sent to Project Griffon. |
+| `timestamp` | Number | Timestamp from the client when the event was sent to Assurance. |
 | `eventNumber` | Number | Used to order when the event was sent. This key is useful when events have the same timestamp. |
-| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.griffon\). |
+| `vendor` | String | Vendor identification string in the reverse domain name format \(for example, com.adobe.assurance\). |
```
Comment (ahmyr) [inline beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md:39]: Same here. Leave it as com.adobe.griffon for now.

File: foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md [hunk 10]
```diff
@@ -156,14 +156,14 @@ D/AdobeExperienceSDK: Assurance - Timeout - Assurance did not receive deeplink t
 
 {% tab title="iOS" %}
 ```text
-[AdobeExperienceSDK DEBUG <AEPAssurance>]: Timeout - Griffon didnot receive deeplink to start griffon session. Shutting down griffon extension
+[AdobeExperienceSDK DEBUG <AEPAssurance>]: Timeout - Assurance did not receive deeplink to start Assurance session. Shutting down Assurance extension
```
Comment (ahmyr) [inline foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md:159]: This is an actual log message from the SDK. Can you check with Prashanth if it says Assurance or still says Griffon? If it says Griffon, we should update the log message. But, want to keep this matching the specific log message for now. Let me know if the log message does still say Griffon and we can create a JIRA ticket to get that updated (and note in the JIRA ticket to also update the doc when the log message is updated).

Comment (ahmyr) [reply foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md:159]: Oh, and, we can see if the "didnot" needs updated in the log message as well.

File: resources/frequently-asked-questions/README.md [hunk 11]
```diff
@@ -24,7 +24,7 @@ There are several new features and benefits of using the Experience Platform Mob
 | Adobe Campaign Standard | Push and in-app messaging | Push only |
 | Adobe Target | ✔️ | ✔️ |
 | Places Service | ✔️ |  |
-| Project Griffon \(BETA\) | ✔️ |  |
+| Adobe Experience Platform Assurance \(BETA\) | ✔️ |  |
```
Comment (ahmyr) [inline resources/frequently-asked-questions/README.md:27]: Take the "BETA" off of this as well :) .

File: beta/project-griffon/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (caseymcj) [reply beta/project-griffon/README.md]: OK, thanks for this note. I removed the beta product warning box.

File: resources/frequently-asked-questions/README.md (hunk 11 above)
Comment (caseymcj) [reply resources/frequently-asked-questions/README.md]: Done! Beta removed.

File: foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md (hunk 10 above)
Comment (caseymcj) [reply foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md:159]: OK, thanks. I sent him a note about this. I'll make any updates to this once I hear back.

File: beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md (hunk 8 above)
Comment (caseymcj) [reply beta/project-griffon/using-project-griffon/custom-validation-and-project-griffon.md]: OK, I reverted this change.

File: beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md (hunk 9 above)
Comment (caseymcj) [reply beta/project-griffon/using-project-griffon/validation-summary-and-project-griffon.md]: Sounds good. Change reverted.

File: foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md (hunk 10 above)
Comment (caseymcj) [reply foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md]: Prashanth was able to help me synchronize all the error logs with what we currently have in the SDKs. 2 images were also updated. This did include fixing the "didnot" as well. :)

**Pull request merged**